docs/paper/paper.md

```diff
@@ -0,0 +1,132 @@
+---
+title: 'Qiskit Experiments: A Python package to characterize and calibrate quantum computers'
+tags:
+  - Python
+  - Quantum computing
+  - Characterization
+  - Calibration
+authors:
+  - name: Naoki Kanazawa
+    orcid: 0000-0002-4192-5558
+    affiliation: 1
+  - name: Daniel J. Egger
+    orcid: 0000-0002-5523-9807
+    corresponding: true
+    affiliation: 2
+  - name: Yael Ben-Haim
+    orcid: 0000-0000-0000-0000
+    affiliation: 3
+  - name: Helena Zhang
+    orcid: 0000-0002-7813-7133
+    affiliation: 4
+  - name: Will E. Shanks
+    orcid: 0000-0002-5045-8808
+    affiliation: 4
+  - name: Gadi Aleksandrowicz
+    orcid: 0000-0000-0000-0000
+    affiliation: 3
+  - name: Christopher J. Wood
+    orcid: 0000-0001-7606-7349
+    affiliation: 4
+affiliations:
+ - name: IBM Quantum – IBM Research Tokyo, Tokyo, 103-8510, Japan
+   index: 1
+ - name: IBM Quantum, IBM Research Europe - Zurich, Ruschlikon 8003, Switzerland
+   index: 2
+ - name: IBM Quantum, IBM Research Israel, Haifa 31905, Israel
+   index: 3
+ - name: IBM Quantum, IBM T.J. Watson Research Center, Yorktown Heights, NY 10598, USA
+   index: 4
+date: 21 February 2023
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Qiskit Experiments is a Python package for designing and running quantum computing experiments 
+with a focus on calibration and characterization of quantum devices. 
+It consists of a general purpose experiments framework which can be used by researchers to rapidly 
+implement new experiments, and a library of common experiments for calibration, characterization, 
+and verification of quantum devices.
+
+The core framework of `Qiskit Experiments` consists of three parts.
+(i) An experiment class defines the quantum circuits to run. 
+(ii) A data container class named `ExperimentData` stores the data measured during the execution 
+of the quantum circuits.
+(iii) An analysis class, attached to each experiment, defines how to analyze the measured data.
+The analysis also attaches its results, such as fit results and figures, to the data container.
+\autoref{fig:framework} summarizes this framework.
+Importantly, this framework can interface with services to store, load, and share data.
+The library of experiments includes common quantum computing experiments such as Randomized 
+Benchmarking [@Magesan2011], Quantum state and process tomography [@Banaszek1999], Quantum Volume [@Cross2019], 
```

Comment:
"Quantum state and process tomography" should either have every word capitalized (besides and) or none.

Reply:
```suggestion
Benchmarking [@Magesan2011], Quantum State and Process Tomography [@Banaszek1999], Quantum Volume [@Cross2019], 
```